doc/manual/tutorial.md

```diff
@@ -121,7 +121,7 @@ By default, nickel exports data as JSON. But we can change that using
 the extra parameter `--format yaml` to export as YAML.
 
 ```shell
-nickel -f users.ncl export --export yaml
+nickel -f users.ncl export --format yaml
 ```
```

Comment:
@yannham  One change, I forgot to push, `--export` is not a valid flag.

Reply:
Ah, good catch! Thanks